src/pages/product/Product.jsx

```diff
@@ -0,0 +1,61 @@
+import React from "react";
+import { Navbar } from "../../components/components-index";
+import { useFilterContext } from "../../context/FilterContext";
+import {
+  fastDeliveryHandler,
+  getProductByOffer,
+  getSortedProducts,
+  IncludeOutOfStockHandler,
+  priceRangeHandler,
+  getProductByCategory,
+  getSortedRating,
+} from "../../utils/utils-index";
+import "./Product.css";
+import { ProductDisplay } from "./ProductDisplay";
+import { Sidebar } from "./Sidebar";
+
```

Comment:
Just a suggestion to import common files in a single file and then export it from that file thereby reducing the import statements in each files.